token_server/app.yaml

```diff
@@ -1 +1 @@
-runtime: nodejs12
+runtime: nodejs10
```

Comment:
Maybe add a todo to update to 12 if/when amazon supports?

Reply:
Done.